content/responses/metadata_query_result.yml

```diff
@@ -9,21 +9,17 @@ description: |-
   A single file or folder that is returned for a metadata query, including its
   metadata that matched the query.
 
-properties:
-  item:
-    allOf:
-      - description: |-
-          A standard representation of a file or folder that matched the search
-          query.
-      - oneOf:
-          - $ref: '#/components/schemas/File--Standard'
-          - $ref: '#/components/schemas/Folder--Standard'
-
-  metadata:
-    type: object
+allOf:
+  - description: |-
+      A standard representation of a file or folder that matched the search
+      query plus optional metadata.
+  - oneOf:
```

Comment:
Actually, upon consideration, I think we can actually remove the whole `MetadataQueryResult` object and just change this line to a `oneOf` of file and folder.

https://github.com/box/box-openapi/blob/default/content/responses/metadata_query_results.yml#L17-L18

Reply:
done in new commit